src/components/Pickers/Suggestions/Suggestions.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import { Button, ButtonType } from '../../Button';
+import { css } from '../../../utilities/css';
+import { ISuggestionItemProps, ISuggestionsProps } from './Suggestions.Props';
+import { ISuggestionModel } from './SuggestionsController';
+import { BaseComponent } from '../../../common/BaseComponent';
+import './Suggestions.scss';
+
+export class SuggestionsItem<T> extends React.Component<ISuggestionItemProps<T>, {}> {
+  public render() {
+    let {
+      suggestionModel,
+      RenderSuggestion,
+      onClick,
+      className
+    } = this.props;
+    return (
+      <Button
+        onClick={ onClick }
+        className={ css('ms-Suggestions-item', { 'is-suggested': suggestionModel.isSelected }, className) }
+        >
+        <RenderSuggestion { ...suggestionModel.item }/>
+      </Button>
+    );
+  }
+}
+
+export class Suggestions<T> extends BaseComponent<ISuggestionsProps<T>, {}> {
+
+  protected _searchForMoreButton: Button;
+  protected _selectedElement: HTMLDivElement;
+  private SuggestionItemOfProperType = SuggestionsItem as new (props: ISuggestionItemProps<T>) => SuggestionsItem<T>;
+
+  constructor(suggestionsProps: ISuggestionsProps<T>) {
+    super(suggestionsProps);
+    this._getMoreResults = this._getMoreResults.bind(this);
+  }
+
+  public render() {
+    let {
+      suggestionsHeaderText,
+      searchForMoreText,
+      className,
+      moreSuggestionsAvailable
+    } = this.props;
+
+    return (
+      <div className={ css('ms-Suggestions', className ? className : '') }>
+        { suggestionsHeaderText ?
+          (<div className='ms-Suggestions-title'>
+            { suggestionsHeaderText }
+          </div>) : (null) }
+        <div className='ms-Suggestions-container' id='suggestion-list' role='menu'>
+          { this._renderSuggestions() }
+        </div>
+        { searchForMoreText && moreSuggestionsAvailable ?
+          (<Button
+            onClick={ this._getMoreResults.bind(this) }
+            className={ 'ms-SearchMore-button' }
+            buttonType={ ButtonType.icon }
+            icon={ 'Search' }
+            ref={ this._resolveRef('_searchForMoreButton') } >
+            { searchForMoreText }
+          </Button>) : (null)
+        }
+      </div>
+    );
+  }
+
+  public focusSearchForMoreButton() {
```

Comment:
...this is why you might want a class interface to expose the public contract in docs.